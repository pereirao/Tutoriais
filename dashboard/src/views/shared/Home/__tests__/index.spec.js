import React from 'react';

// NOTE: the named export isn't a connected component
import { Home } from '../index';

it('renders when logged_in', () => {
  const stubbed_location = {pathname: '/'};
  expect(shallow(
    <Home
      logged_in
      fetching_me={false}
      location={stubbed_location}
      fetchMe={() => {}} />
  )).toMatchSnapshot();
});

it('get loading view when !logged_in', () => {
  const stubbed_location = {pathname: '/'};
  expect(shallow(
    <Home
      logged_in={false}
      fetching_me={false}
      location={stubbed_location}
      fetchMe={() => {}} />
  )).toMatchSnapshot();
});
