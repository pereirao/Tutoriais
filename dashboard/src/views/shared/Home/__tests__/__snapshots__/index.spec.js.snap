// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get loading view when !logged_in 1`] = `
<div
  className="loaderRow"
>
  <MBLoader
    size="large"
    type="spinner"
  />
</div>
`;

exports[`renders when logged_in 1`] = `
<Redirect
  push={false}
  to={
    Object {
      "pathname": "/orders/active",
      "state": Object {
        "from": Object {
          "pathname": "/",
        },
      },
    }
  }
/>
`;
